frontend/src/pages/DataCatalog/MethodologyTab.tsx

```diff
@@ -348,55 +348,143 @@ function MethodologyTab() {
+                  the CAWP website. All gender and race/ethnicity
                   categorizations are self-reported, and a legislator may be
                   represented in multiple race groupings if that is how they
                   identify.
                 </p>
 
+                <p>
+                  We are also able to track these rates over time (currently
+                  only for U.S. Congress) as outlined below. A member is counted
+                  towards any year in which they served even a single day. This
+                  results in the "bumpiness" observed as the proportions change
+                  incrementally between election and non-election years. We only
+                  track back to just before the first woman was elected to the
+                  U.S. Congress in 1917.
+                </p>
+                <ul>
+                  <li>
+                    Historical, intersectional representation (e.g.{" "}
+                    <i>
+                      “In each year since 1915, what percent of all U.S.
+                      Congress members identified as black women?”
+                    </i>
+                    ). We obtain the historic counts of U.S. Congress members,
+                    by year and by state/territory, from the open-source{" "}
+                    <a href={urlMap.unitedStatesIo}>@unitedstates project</a>.
+                  </li>
+                  <li>
+                    Historical relative inequity (e.g.{" "}
+                    <i>
+                      “In each year since 2019, what percent over- or
+                      under-represented were black women when compared to their
+                      share of represention amongst all women Congress members?”
+                    </i>
+                    ) Note: we currently track this measure back only to 2019,
+                    as we are utilizing the 2019 ACS 5-year estimates for the
+                    population comparison metric.{" "}
+                  </li>
+                </ul>
+
+                <p>
+                  Unfortunately CAWP and the U.S. Census use some different
+                  race/ethnicity groupings, making direct comparisons and
+                  calculations difficult or impossible in some cases. For
+                  specific methodology on the race groups collected by CAWP,
+                  please <a href={urlMap.cawp}>visit their database directly</a>{" "}
+                  . We have made several adjustments to our methods to
+                  incorporate these non-standard race groupings when possible:
+                </p>
+
+                <ul>
+                  <li>
+                    Women who identify as multiple specific races are listed
+                    multiple times in each corresponding race visualization.
+                    Therefor, these race/ethnicity groupings are non-exclusive,
```

Comment:
Therefore*

Reply:
nice catch! it was using the wrong spelling in 3 other locations in our code as well haha